docs/deployment.md

```diff
@@ -35,8 +35,8 @@ This is the recommended method if you provisioned the servers using the supplied
 
 [inventories/openstack-idr.py](../inventories/openstack-idr.py) is a modified version of the [default Ansible OpenStack dynamic inventory](https://raw.githubusercontent.com/ansible/ansible/devel/contrib/inventory/openstack.py) to make it easier to use an [Ansible SSH gateway/jump host](http://docs.ansible.com/ansible/faq.html#how-do-i-configure-a-jump-host-to-access-servers-that-i-have-no-direct-access-to), and to configure multiple networks.
 The main changes are:
-- automatic setting of SSH ProxyCommand
-- ordering of network interfaces
+- Automatic setting of SSH ProxyCommand.
+- Ordering of network interfaces.
```

Comment:
True, but I think adding `;` and `.` makes things harder to read in the context of a bulleted list https://github.com/IDR/deployment/pull/8#discussion_r101546420